Task 6/Task6.1/main.css

```diff
@@ -0,0 +1,22 @@
+body {
+    font-family: 'Roboto Slab', serif;
+    font-size: 12pt;
+}
+
+.menu {
+    width: 200px;
+    padding: 0;
+    margin: 0;
+    list-style-type: none;
+    background-color: burlywood;
+    color: saddlebrown;
+    box-sizing: border-box;
+    display: inline-block;
+    box-shadow: 5px 5px 0 2px saddlebrown;
+    border-radius: 5px;
+}
+
+.menu>li {
+    padding-top: 10px;
+}
```

Comment:
Missing another selector for `:first-child` to remove this margin where it's not needed (additional margin at the top, if make padding on menu element the top)